trakt/movies.py

```diff
@@ -343,6 +344,31 @@ def scrobble(self, progress, app_version, app_date):
         """
         return Scrobbler(self, progress, app_version, app_date)
 
+    def checkin(self, app_version, app_date, message="", sharing={},
```

Comment:
`sharing` should also default to `None` here as well

Reply:
done